Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -87,11 +99,44 @@ - (OCTCall *)callToChat:(OCTChat *)chat enableAudio:(BOOL)enableAudio enableVide
     return nil;
 }
 
+- (BOOL)enableVideoSending:(BOOL)enable forCall:(OCTCall *)call error:(NSError **)error
+{
+    OCTToxAVVideoBitRate bitrate = (enable) ? kDefaultVideoBitRate : kOCTToxAVVideoBitRateDisable;
+    BOOL success = [self setVideoBitrate:bitrate forCall:call error:error];
```

Comment:
I think when you set it to 0, toxav notifies the friend that you stopped sending video. That way the other user can update the UI appropriately.